src/libs/analytics.ts

```diff
@@ -18,6 +18,7 @@ enum AnalyticsEvent {
   CONSULT_WHOLE_OFFER = 'ConsultWholeOffer',
   SHARE_OFFER = 'Share',
   WHY_ANNIVERSARY_CLICKED = 'SignUp-whyAnniversary',
+  CANCEL_SIGNUP = 'SignUp-cancelSignUp',
```

Comment:
https://firebase.google.com/docs/reference/cpp/group/event-names#:~:text=Event%20names%20can%20be%20up,and%20should%20not%20be%20used.

> Event names can be up to 40 characters long, may only contain alphanumeric characters and underscores ("_"), and must start with an alphabetic character. The "firebase_", "google_", and "ga_" prefixes are reserved and should not be used.